components/style/values/computed/pointing.rs

```diff
@@ -0,0 +1,73 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+//! Computed values for Pointing properties.
+//!
+//! https://drafts.csswg.org/css-ui/#pointing-keyboard
+
+#[cfg(feature = "gecko")]
+use std::fmt;
+#[cfg(feature = "gecko")]
+use style_traits::ToCss;
+use style_traits::cursor::CursorKind;
+#[cfg(feature = "gecko")]
+use values::specified::url::SpecifiedUrl;
+
+/// The computed value for the `cursor` property.
+/// Servo variant.
+/// https://drafts.csswg.org/css-ui/#cursor
+#[cfg(not(feature = "gecko"))]
```

Comment:
How about just using `#[cfg(feature = "servo")]`?
Maybe we don't need the `Servo variant` comment with this config attribute.

Reply:
It's a bit different.
When we build with `./mach build-geckolib` we trigger the `#[cfg(feature = "gecko")]` feature and compile in all code marked with it. If we mark that part with `#[cfg(feature = "servo")]`, then it will be compiled in with `./mach build-geckolib`, which we don't want. However, marking it with `#[cfg(not(feature = "gecko"))]` allows us to exclude it in when we compile with `./mach build-geckolib`, which is what we want.

Reply:
Oh, in my understanding, if we mark it as `#[cfg(feature = "servo")]`, it will only be compiled when `feature` is `servo` so it will be ignored when `feature` is `gecko`.

Reply:
Oh, wait, I think you're right actually. I thought for some reason that it will be compiled when `feature` is `servo`, which it won't. I need to get some sleep haha.

Reply:
Sleep tight! and thanks for your confirmation :)

Reply:
No problem :). I just thought there could be a possibility that when we compile with `./mach build-geckolib` both `servo` and `gecko` `feature` is enabled. Let's wait what Emilio says :).

Reply:
Sure! I'm looking forward to seeing his comment too!

Reply:
Yeah, `feature = "gecko"` and `feature = "servo"` are mutually exclusive. So `not(feature = "gecko")` and `feature = "servo"` are effectively the same. Use whatever you please :)